viz/visualize.py

```diff
@@ -33,15 +35,22 @@ def __init__(self, model, dataset, model_dir=None, save_images=True):
 
         dataset : str
             Name of the dataset.
+
+        loss_of_interest : str
+            The loss type (as saved in the log file) to order the latent dimensions by and display (optionally)
+
+        display_loss_per_dim : bool
+            if the loss should be included as text next to the corresponding latent dimension images.
         """
         self.model = model
         self.device = next(self.model.parameters()).device
         self.latent_traverser = LatentTraverser(self.model.latent_dim)
         self.save_images = save_images
         self.model_dir = model_dir
         self.dataset = dataset
+        self.loss_of_interest = loss_of_interest
 
-    def show_disentanglement_fig2(self, reconstruction_data, latent_sweep_data, heat_map_data, latent_order=None, heat_map_size=(32, 32), filename='imgs/show_disentanglement.png', size=8, sample_latent_space=None):
+    def show_disentanglement_fig2(self, reconstruction_data, latent_sweep_data, heat_map_data, latent_order=None, heat_map_size=(32, 32), filename='show_disentanglement.png', size=8, sample_latent_space=None):
```

Comment:
Please write this def over multiple lines so it can all be seen.

Reply:
I'll do this but I am expecting Bart to open a PR on this very soon